src/source/shift.rs

```diff
@@ -65,6 +65,7 @@ impl Shift {
 
         debug!(target: "Shift", "Launching script {}", script);
         let mut child = Command::new(&*self.bin_path)
+            .arg("--max_old_space_size=4096")
```

Comment:
Ok, this one is the interesting module.

Ideally, we should be able to customize this value without changing the code, but this can wait for a followup patch.

Could you just add a comment explaining why we're doing this?